svgpath2mpl.py

```diff
@@ -27,19 +27,30 @@
 
 COMMAND_RE = re.compile(r"([MLHVCSQTAZ])([^MLHVCSQTAZ]*)", re.IGNORECASE)
 FLOAT_RE = re.compile(r"[-+]?[0-9]*\.?[0-9]+(?:[eE][-+]?[0-9]+)?")
-COMMANDS = { 
+COMMANDS = {
     'M' : (Path.MOVETO,),    # moveto
     'L' : (Path.LINETO,),    # line
     'H' : (Path.LINETO,),    # shorthand for horizontal line
     'V' : (Path.LINETO,),    # shorthand for vertical line
     'Q' : (Path.CURVE3,)*2,  # quadratic bezier
     'T' : (Path.CURVE3,)*2,  # shorthand for smooth quadratic bezier
     'C' : (Path.CURVE4,)*3,  # cubic bezier
-    'S' : (Path.CURVE4,)*3,  # shorthand for smooth cubic bezier 
+    'S' : (Path.CURVE4,)*3,  # shorthand for smooth cubic bezier
     'Z' : (Path.CLOSEPOLY,), # closepath
     'A' : None               # arc
 }
-
+PARAMS = {
+    'M' : 2, # moveto
+    'L' : 2, # line
+    'H' : 1, # shorthand for horizontal line
+    'V' : 1, # shorthand for vertical line
+    'Q' : 4, # quadratic bezier
+    'T' : 4, # shorthand for smooth quadratic bezier
+    'C' : 6, # cubic bezier
+    'S' : 6, # shorthand for smooth cubic bezier
+    'Z' : 0, # closepath
+    'A' : 0  # arc
```

Comment:
I think the problem is here. The number of arc instructions is dynamically determined.